test/integration/targets/win_pester/tasks/main.yml

```diff
@@ -0,0 +1,117 @@
+---
+- name: Get facts
+  setup:
+
+- name: Add Pester module
```

Comment:
I would consider adding these steps to the examples so users can know how to ensure they have pre-requisites for running the module.

Reply:
Sure thing !

Reply:
Done !